### DISCUSSION TITLE
Can I execute a .NET (Core) console application?

### DISCUSSION THREAD

**Discussion by ajtatum** [jhuckaby/Cronicle#706] [Q&A] [answered]
I know it says any language, but wanted to check specifically about .NET 8 console applications. I would write them to accept JSON for input and would write JSON to the console after it's done.

Thanks,
AJ

**Comment by dsturgilljr** [comment, marked as answer]
Yes! We have written a .NET console app that is published as a self-contained executable as part of our Cronicle build. Our Cronicle plugins call our console app, and we use the parameters of Cronicle to pass in information for different functions.

**Reply by ajtatum** [reply]
> Yes! We have written a .NET console app that is published as a self-contained executable as part of our Cronicle build. Our Cronicle plugins call our console app, and we use the parameters of Cronicle to pass in information for different functions.

That is awesome to hear! Thank you for replying. It seems like this can be tremendously beneficial and save me a lot of time down the road!
